### ISSUE TITLE
Error when unpacking with current head

### ISSUE THREAD

**Issue by nuschpl** [platomav/MEAnalyzer#54]
I've got the following. If this is relevant I'm launching mea.py via a symlink from different path, it doesn't cause problem in other operations however with the 'out' is my output prefix I see it doubled

```bash
Error: ME Analyzer v1.304.5 crashed, please report the following:

Traceback (most recent call last):
  File "/Users/CUT/MEA.py", line 13012, in <module>
    cse_unpack(variant, fpt_part_all, bpdt_part_all, file_end, fpt_start if rgn_exist else -1, fpt_chk_fail, cse_lt_chk_fail,
  File "/Users/CUT/MEA.py", line 5808, in cse_unpack
    rbe_pm_met_valid = mod_anl(cpd_offset_e, cpd_mod_attr_e, cpd_ext_attr_e, fw_name, ext_print, ext_phval, ext_dnx_val, ext_iunit_val,
  File "/Users/CUT/MEA.py", line 7017, in mod_anl
    with open(gmf_cert_path, 'wb') as gmf_cert : gmf_cert.write(gmf_blob_info[3][0])
FileNotFoundError: [Errno 2] No such file or directory: 'out/out/Unpacked_CUT_SPI.bin/RBEP 0001 [0x104000]/GMF_Certificate.crt'
```

**Comment by nuschpl** [comment]
I've fixed this be createing 'out' directory manually and:
```bash
cd out
ln -s . out
``` 
within it. This solved the problem with subsequent call to MEAnalyser which was putting some files in 'out' and some in 'out/out' then search appropriate file in location it wasn't present.

**Comment by platomav** [comment]
I don't understand what is the problem. MEA does not create an "out" folder. What is the exact command given to MEA (not shown in the log)? From the 2nd reply, it would appear that you have some strange folder structure with symlinks etc.

**Comment by nuschpl** [comment]
The command is 
`./MEA.py Unpacked_CUT_SPI.bin -skip -exit -dfpt -unp86 -bug86 -out out`
when the signle out dir is present in `pwd`  the tool creates some files directly in it and some files by creating another `out` in existing `out`. Then it searches in `out/out` files which it previously created in `out`.

**Comment by platomav** [comment]
Interesting, I can confirm this behavior is a bug. Thanks for the report. That is a ~4.5y old bug which got overlooked because it was not causing problems before MEA v1.300.0 (when the custom output directory capability was added). I can fix it easily and push a new MEA version.

**Comment by platomav** [comment]
The issue should now be fixed via [a3bc824](https://github.com/platomav/MEAnalyzer/commit/a3bc82427ba8198260db9db4efec4aca4eaf8684). Could you check again?

**Comment by Jjajsja** [comment]
https://github.com/platomav/MEAnalyzer/blob/master/Changelog.txt
